src/content/events/2023/hackathon-october-2023.mdx

```diff
@@ -0,0 +1,240 @@
+Please see the [Summit website Hackathon Agenda](https://summit.nextflow.io/barcelona/agenda/hackathon/#events) for details.
+Note that there is a [calendar view](https://summit.nextflow.io/barcelona/agenda/calendar/)
+with links that you can use to subscribe / add to your personal calendar.
+
+## Venue: Hotel SB Glow
+
+The in-person hackathon will be taking place at [Hotel SB Glow](https://goo.gl/maps/6uKCKkbciLAR5qgSA).
+More information about this venue can be found the the [Nextflow Summit website](https://summit.nextflow.io/).
+
+# Groups
+
+To keep things manageable everyone will be organized into groups.
+You are free to change groups at any time during the hackathon.
+
+We will coordinate our work and the issues we are working on using a single [GitHub project board](https://github.com/orgs/nf-core/projects/47).
+There is a separate tab for each group that will have a list of relevant issues for you to work on.
+
+## Hackathon focus: nf-test
+
+At this hackathon, our special topic is [nf-test](https://code.askimed.com/nf-test/) 🚀
+
+nf-test is a testing framework for Nextflow code.
+We want to start using it for all nf-core pipelines and modules.
+Some proof-of-concept work has already been done, with standardised implementation
+patterns mostly worked out, but there's still a lot of work to add this in for all components.
+
+There are nf-test related groups in several of the groups below.
+It would be great to move nf-test adoption forward during the event!
+
+## Meta groups
+
+These groups are here to support anyone who may need help getting started at the hackathon.
+They are not intended for focused task-specific work.
+
+<HackathonGroup title="Beginners" leaders={{
+    'mribeirodantas': 'Marcel Ribeiro-Dantas',
+    'abhi18av': 'Abhinav Sharma',
+    'FranBonath': 'Franziska Bonath'
+}}>
+    For this hackathon, we will  have no designated beginners group but will instead
+    have a "beginners table" **on the first day**.
+    You're welcome to join to get help on how to contribute to open-source,
+    and to meet other community contribution beginners.
+
+    Please note that attendees are expected to know the basics of building with Nextflow.
+
+</HackathonGroup>
+
+<HackathonGroup
+    title="Online"
+    leaders={{
+        yuukiiwa: 'Yuk Kei',
+        louperelo: 'Louisa Perelo',
+        abhi18av: 'Abhinav Sharma',
+    }}
+>
+    If you're feeling lost in Gather.town, or just want to hang out with other online attendees, look out for Yuk Kei
+    and Louisa. They'll be able to steer you in the right direction and get you started with one of the project groups.
+</HackathonGroup>
+
+## Project groups
+
+<HackathonGroup title="Pipelines" color="#bf4b8a" proj_url="https://github.com/orgs/nf-core/projects/47/views/4" leaders={{
+    'drpatelh': 'Harshil Patel',
+    'adamrtalbot': 'Adam Talbot',
+    'MaxUlysse': 'Maxime Garcia',
+    'Emiller88': 'Edmund Miller',
+    'sateeshperi': 'Sateesh Peri',
+}}>
```

Comment:
```suggestion
<HackathonGroup title="Pipelines" color="#bf4b8a" proj_url="https://github.com/orgs/nf-core/projects/47/views/4" leaders={{
    'drpatelh': 'Harshil Patel',
    'adamrtalbot': 'Adam Talbot',
    'MaxUlysse': 'Maxime Garcia',
    'Emiller88': 'Edmund Miller',
    'sateeshperi': 'Sateesh Peri',
}} slack=https://nfcore.slack.com/archives/C05RYK1F4PP>
```

Reply:
Sorry, should have said - `slack` here is literally the Slack channel name, not the URL. Also needs to be in quotes.